source/appendices/e-audit-requirements-and-tests/02-general-tests.rst

```diff
@@ -0,0 +1,85 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.2 General Audit Tests
+-----------------------
+
+This section of the appendix outlines tests that MUST be run during all audits, and which are not specific to a particular COUNTER Report.
+
+
+E.2.1 Validation Tool
+"""""""""""""""""""""
+
+Once reports are available for testing (i.e. up to 28 days after the end of the seeding month), auditors MUST run the COUNTER Reports and Standard Views of COUNTER Reports that are within the scope of the audit through the Validation Tool. This test must include all attributes report providers are required to support, including attributes that are included only if called for.
+
+Where report providers have elected to follow the pre-flight preparation step outlined in Section 9 of the Code of Practice, this audit test should not result in any errors.
+
+
+E.2.2 SUSHI Endpoints
+"""""""""""""""""""""
+
+The auditor MUST test the SUSHI service endpoints as follows
+
+* The /status endpoint MUST be public (i.e. unprotected) to allow report consumers to easily check whether the service is live.
+* The /reports endpoint MUST return the date range for which COUNTER Reports and the Standard Views of COUNTER Reports are available. If this is less than the required year-to-date plus two years of back data, the auditor MUST note the missing months in the audit report.
+* The /members endpoing MUST return the auditor's customer details.
+
+Any error messages returned by the SUSHI service in the process of audit MUST be noted in the audit report, including an indication of where these error messages deviate from those specified in  :ref:`Appendix D <appendix-d>`.
+
+
+E.2.3 Audit Tests for Double-Click Filtering
```

Comment:
The number are correct when whole book downloads which identifiable segments can be avoided. In case a content provider only has books on the platform, offers only whole book downloads and can identify segments, this isn't possible.